libs/mock-block-dock/dev/dev-app.tsx

```diff
@@ -40,12 +40,20 @@ const blockEntityMap = {
 
 const DevApp: FunctionComponent = () => {
   const [testBlockType, setTestBlockType] = useState<TestBlockType>("react");
+  const [blockProtocolApiKey, setBlockProtocolApiKey] = useState<string>();
```

Comment:
Why does this need to be in state?

Reply:
i.e. you could just pass `process.env.BLOCK_PROTOCOL_API_KEY` directly as a prop for the same effect

Reply:
Good point, have addressed in https://github.com/blockprotocol/blockprotocol/pull/966/commits/d1f2423c0b696dbcaa34ad525dc298df3e415475